pkgs/development/libraries/libtiff/default.nix

```diff
@@ -92,5 +92,6 @@ stdenv.mkDerivation (finalAttrs: {
     license = licenses.libtiff;
     platforms = platforms.unix;
     pkgConfigModules = [ "libtiff-4" ];
+    maintainers = teams.geospatial.members;
```

Comment:
Tiff is not specific to GIS so this is not ideal, but as there was no maintainer before, better adopt it yes. Let's hope more maintainers steps in.

Reply:
libtiff is already maintained by [us](https://github.com/NixOS/nixpkgs/blob/595e00cbcac55249c2f27998472210b71c129227/pkgs/by-name/li/libtiff/package.nix#L107) :).